### ISSUE TITLE
SymbolSource built-in hosting

### ISSUE THREAD

**Issue by FeodorFitsner** [appveyor/ci#638]
Discussion: http://help.appveyor.com/discussions/questions/2089-publishing-nuget-symbols-to-a-private-source

**Comment by jezzsantos** [comment]
Hey @FeodorFitsner,
Nothing has changed in the last few months has it?
We still don't have Nuget symbols on AppVeyor. Is that correct? MyGet or SymbolSource.org are the current options, right?

**Comment by FeodorFitsner** [comment]
MyGet is a great choice for hosting symbols. We might implement symbols hosting in the future, but right now it's not a highest priority.

**Comment by greigs** [comment]
The documentation seems to suggest this has been implemented. Is this correct?https://www.appveyor.com/docs/nuget/#automatic-publishing-of-nuget-projects

**Comment by FeodorFitsner** [comment]
Nope, AppVeyor doesn't provide symbols hosting (understood by Visual Studio). Symbol packages have `.pdb` files inside and some teams prefer to have them on feed instead of "release" packages.

**Comment by bazzilic** [comment]
Any updates on this?

**Comment by the-avid-engineer** [comment]
This would be a really great feature, 10/10 would use

**Comment by kpalm** [comment]
Please implement this feature
